proteus/FemTools.py

```diff
@@ -3998,6 +4083,16 @@ def writeVectorFunctionXdmf(self,ar,uList,components,vectorName,tCount=0,init=Tr
         self.XdmfWriter.writeVectorFunctionXdmf_nodal(ar,uList,components,vectorName,"c0p2_Lagrange",tCount=tCount,init=init)
 
 
+# Lagrange Factory On Cube
+def LagrangeCubeFactory(OrderIn):
+    class LagrangeCubeOrderN(C0_AffineLagrangeOnCubeWithNodalBasis):
+        def __init__(self,mesh,nd):
+            C0_AffineLagrangeOnCubeWithNodalBasis.__init__(self,mesh,nd,order=OrderIn)
+    return LagrangeCubeOrderN
+
+# Q1 = LagrangeCubeFactory(1)
```

Comment:
We could just to `Q1 = C0_AffineLagrangeOnCubeWithNodalBasis` to make example code cleaner.